x-pack/plugins/monitoring/public/angular/app_modules.ts

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import angular, { IWindowService } from 'angular';
+import '../views/all';
+// required for `ngSanitize` angular module
+import 'angular-sanitize';
+import 'angular-route';
+import '../index.scss';
```

Comment:
Consider breaking this up (in a later PR). You can now import the individual sass files directly into the component themselves `import './my_component.scss`. Otherwise the entirety of the the css loads at once. Just helps with performance.

Reply:
Will do, Thank you!